### PR TITLE
Fix warning for the declaration of wpSetUpBeforeClass( $factory )

### PR DESCRIPTION
**Pull request by Chouby**
Required since WP 5.6. See https://core.trac.wordpress.org/ticket/51344

### CODE REVIEW

**Review by raaaahman — COMMENTED** [review]
Are you sure this is mandatory? The PR you quoted as an example don't have it. ;thinking_face: https://github.com/WordPress/wordpress-develop/pull/643/files#

File: tests/phpunit/includes/testcase-trait.php [hunk 1]
```diff
@@ -16,7 +16,7 @@ trait PLL_UnitTestCase_Trait {
 	 *
 	 * @param WP_UnitTest_Factory $factory
 	 */
-	static function wpSetUpBeforeClass( $factory ) { // phpcs:ignore WordPress.NamingConventions.ValidFunctionName.MethodNameInvalid
+	static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) { // phpcs:ignore WordPress.NamingConventions.ValidFunctionName.MethodNameInvalid
```
[the remark this answers is not in the document]
Comment (Chouby) [reply tests/phpunit/includes/testcase-trait.php:19]: Yes. CamelCase is not valid in WordPress coding standards. As far as I know, there is no way to globally whitelist some function names such as this one.

**Comment by Chouby** [comment]
> Are you sure this is mandatory? The PR you quoted as an example don't have it. :thinking_face: https://github.com/WordPress/wordpress-develop/pull/643/files#

You did not look at the right place. Here is the right changeset: https://core.trac.wordpress.org/changeset/49603

File: tests/phpunit/includes/testcase-trait.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (raaaahman) [reply tests/phpunit/includes/testcase-trait.php:19]: I get it. Although, it seems to be the only one place where we need to tell PHPCS to ignore it ...

Could it be linked to the method's signature that doesn't include the `public` keyword? (I only see this difference with the other `wpSetUpBeforeClass()` declarations).

[the remark this answers is not in the document]
Comment (Chouby) [reply tests/phpunit/includes/testcase-trait.php:19]: No. My guess is that the difference is that we are in a trait. In other cases, we are in classes extending a parent class already  defining this method. Maybe the snif is smart enought to accept this case.

**Review by raaaahman — APPROVED** [review]

**Pull request merged**